Markdown_HW1_IzzyGaw.Rmd

```diff
@@ -0,0 +1,20 @@
+---
+title: "R Markdown Super Awesome Homework 1"
+author: "Izzy Gaw"
+date: "2020-01-16"
+output: html_document
+---
+
+This is a text paragraph in an R Markdown document.
```

Comment:
Great job inserting your R code below. However, can you please replace the example text with your explanations?
Remember, effective documentation is concise :)